src/utils/versions/parseNodesToVersionsValues.ts

```diff
@@ -18,15 +20,16 @@ export const parseNodesToVersionsValues = (
         }
         return acc;
     }, {});
-
-    return Object.keys(versionsCount).map((version) => {
+    const result = Object.keys(versionsCount).map((version) => {
+        const value = (versionsCount[version] / nodes.length) * 100;
         return {
             title: version,
             version: version,
             color: versionsToColor?.get(getMinorVersion(version)),
-            value: (versionsCount[version] / nodes.length) * 100,
+            value: value < MIN_VALUE ? MIN_VALUE : value,
```

Comment:
Why MIN_VALUE is 0.5 ?

Reply:
It was kinda a shot in the dark. It looks nice - not wide, but sufficient to hove a mouse.